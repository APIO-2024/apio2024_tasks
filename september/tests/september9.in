txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 5
0 0 2 2 3 1
4 5 6 3 1 2
4 5 6 1 3 2
4 5 6 3 1 2
4 5 6 3 1 2
4 5 6 3 1 2
10 5
0 1 1 2 4 5 0 2 0
9 6 5 7 8 4 3 1 2
9 6 7 5 4 8 1 2 3
6 9 7 5 4 8 2 1 3
6 9 7 5 4 2 1 8 3
6 9 7 5 4 2 1 8 3
7 5
0 0 0 3 1 4
6 5 4 1 3 2
5 6 1 4 3 2
5 6 1 3 4 2
5 1 6 4 2 3
5 1 4 2 6 3
6 5
0 0 2 0 2
4 1 5 2 3
4 5 1 2 3
5 4 2 3 1
5 4 2 1 3
5 4 1 3 2
