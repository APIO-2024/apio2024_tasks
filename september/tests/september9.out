7ckgnn4wyi495puj3ibqf81dqvapyv6b
5
3
1
1
